packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js

```diff
@@ -26,6 +26,54 @@ export const square = {
         float canStroke = smoothstep(vSize - 2.0, vSize, distance * vSize);`
 };
 
+
+//    |
```

Comment:
I can't decipher this diagram, maybe I need 3D glasses?!

Reply:
Added comments